src/components/Rulebook.js

```diff
@@ -0,0 +1,20 @@
+import { Rules } from "./Rules"
+
+export const Rulebook = () => {
+
+
+  return (
+    <article className="rulebook">
+      <ul className="rules">
+        <h2>Talents</h2>
+        <li className="talent">
+          <Rules />
+        </li>
+      </ul>
+    </article>
+  )
+}
+
+
+
+
```

Comment:
bunch of empty lines, but otherwise lovely

Reply:
Nope!! Every line should count! This will be my first fix hehe